README.md

```diff
@@ -7,65 +7,91 @@
 
 # Athens
 
-Athens is a knowledge graph for research and notetaking. Athens is open-source, private, extensible, and community-driven.
+Athens is an open-source, self-hosted app for research and notetaking. Outline your ideas, connect them with links and embedded content, and build a personal knowledge database without having to worry about organizing your thoughts. Athens makes your writing more useful over time by making it easy to find and learn from again.
+
+Athens solves the [big problem](#problem) with information overload while _also_ being open-source, [private](#private), [extensible](#extensible), and [community-driven](#community-driven). We have a much larger [vision](https://athensresearch.gitbook.io/handbook/community/about/vision) for self-hosting, bi-directional links, collective knowledge, and learning in public.
 
 # 🚀 [Download the free desktop app](https://github.com/athensresearch/athens/releases) 🚀
 
-**[Sponsor us on Open Collective](https://opencollective.com/athens) until we migrate to Stripe! We are building out a software service for individuals and teams and a [self-hosted/on-premise solution for enterprises](https://github.com/athensresearch/athens-backend).**
+<details>
+  <summary>Not sure if Athens is right for you?</summary>
+
+  **[Demo Athens in your browser](https://athensresearch.github.io/athens)** to see how it works.
+
+  > ⚠️ The demo doesn't work in Safari due to a limitation in the 
```

Comment:
@joelhans  You forgot to complete the sentence

Reply:
Good catch! I was waiting to see if there was a good explanation for _why_ it doesn't work in Safari, but that appears to be a mystery.